optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/testgen/fact/AbstractValueProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.optaplanner.core.impl.score.director.drools.testgen.fact;
+
+abstract class AbstractValueProvider implements ValueProvider {
```

Comment:
Even though it's not public, it does show in ctrl-n in IntelliJ, so let's prefix all classes with TestGen?

Visibility doesn't really help there. They can be reverted public btw (but don't need to be, it's all good).